### DISCUSSION TITLE
Replacement link for airframes.org in the db editor?

### DISCUSSION THREAD

**Discussion by jaymot123** [vradarserver/vrs#61] [General]
Now that the owner of airframes.org has decided to take his site down, are there any plans for providing a replacement link to another site where lookups can be done?

**Comment by vradarserver** [comment]
That's unfortunate, I wasn't aware that they'd taken the site down.

If you're aware of a reputable site that offers aircraft lookups then let me know. They'd need to let people search for aircraft by supplying an ICAO somewhere in the URL.

**Reply by jaymot123** [reply]
I've found one but it doesn't provide the same information, just the country, tail number, operator and model:  https://opensky-network.org/aircraft-database

**Reply by jaymot123** [reply]
One, airframes.org is back. Two, I found an alternate site that allows searching by ICAO hex code: https://www.mictronics.de/aircraft-database/index.php. Perhaps you could add this second site in case the owner of airframes.org shuts his site down again? It's better to have alternatives and redundancy.
